rs/boundary_node/ic_boundary/src/acme.rs

```diff
@@ -33,6 +33,7 @@ pub trait Ready: Sync + Send {
 
 #[derive(Debug, thiserror::Error)]
 pub enum FinalizeError {
+    #[allow(dead_code)]
```

Comment:
Why is this needed? From the first glance it looks like the code is [used](https://sourcegraph.com/github.com/dfinity/ic/-/blob/rs/boundary_node/ic_boundary/src/acme.rs?L141)?

Reply:
There's a lot of `#[allow(dead_code)]` in `/rs/boundary_node` for code that is actually used. Cargo clippy complains that the code is dead, and I assume it's due to some feature gates that aren't correctly configured in `Cargo.toml`. But I am not sure, and haven't been able to figure out the root cause.

Reply:
Gotcha, let's fix it on the proper level before merging, i could help later (got sick now)

Reply:
Do you mind if the @dfinity/boundary-node team rather creates a ticket and cleans up the code you own in a follow up MR, to not add more noise to the already gigantic MR?

Reply:
We agreed to fix this in follow up PRs